repo/packages/H/hdfs/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pip": [
+    "git+https://github.com/mesosphere/dcos-hdfs.git#dcos-hdfs=0.1.0"
```

Comment:
Unfortunately the URI above does not enforce that version 0.1.0 is being installed. It is just installing what is in master. If you want to do that you can create a tag in the repo for this 0.1.0 public release and do the following: `"git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0`